src/Type/Php/StrSplitFunctionReturnTypeExtension.php

```diff
@@ -101,7 +101,7 @@ public function getTypeFromFunctionCall(FunctionReflection $functionReflection,
 		$items = isset($encoding)
 			? mb_str_split($stringValue, $splitLength, $encoding)
 			: str_split($stringValue, $splitLength);
-		if (!is_array($items)) {
+		if ($items === false) {
```

Comment:
because of https://github.com/phpstan/phpstan-src/commit/119facc64db3acd77b8024fafa8bb3228e90cb25 doing a proper check here which works across php versions is hard.

Reply:
maybe you can use e.g. ```/* @phpstan-ignore-line | different result for PHP 8 */``` here?

Reply:
The only way to get rid of this is to ignore it in a specific file: https://github.com/phpstan/phpstan-src/tree/master/build